packages/strapi-generate-new/files/config/environments/production/database.json

```diff
@@ -10,7 +10,8 @@
         "port": "${process.env.DATABASE_PORT || 27017}",
         "database": "${process.env.DATABASE_NAME || 'production'}",
         "username": "${process.env.DATABASE_USERNAME || ''}",
-        "password": "${process.env.DATABASE_PASSWORD || ''}"
+        "password": "${process.env.DATABASE_PASSWORD || ''}",
+        "ssl": false
```

Comment:
Maybe it's could be a cool to also have an environment variable for this setting ?